.github/workflows/test-rust.yaml

```
@@ -0,0 +1,76 @@
+name: test-rust
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    paths:
+      - ".github/workflows/test-rust.yaml"
+  workflow_call:
+    inputs:
+      os:
+        required: false
+        default: ubuntu-latest
+        type: string
+      target_option:
+        required: false
+        default: --target=wasm32-unknown-unknown
```

```suggestion
        default: ""
```

(and to the extent there was a description of this field in the original, would be great if we can copy it here as a comment)

---

Can I ask more context about which comments? I'm seeing the description of the target-option or this one: 

>  # Without inputs.target, it seems to select linux x64 in the wasm32 tests, though in the docs it says it uses the target as the key?